recipes/spooles/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "spooles" %}
+{% set version = "2.2.0" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}_{{ version }}.tar.gz
+  url: http://www.netlib.org/linalg/spooles/spooles.2.2.tgz
+  sha256: a84559a0e987a1e423055ef4fdf3035d55b65bbe4bf915efaa1a35bef7f8c5dd
+  patches:
+    - gcc.patch
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - rsync  # [unix]
+
+test:
+  commands:
+    - test -f ${PREFIX}/lib/spoolesMT.a  # [unix]
+
+about:
+  home: http://www.netlib.org/linalg/spooles/spooles.2.2.html
+  summary: 'Simple, fast, extensible JSON encoder/decoder for Python'
+  license: LGPL  # the project has no restrictions. aur lists gpl as license
+  license_family: LGPL
```

Comment:
You need to have the licensr_file entry here since this is lgpl

Reply:
The aur-package lists gpl: https://aur.archlinux.org/packages/spooles/

Reply:
That is weird, because arch also says it is public domain: https://aur.archlinux.org/cgit/aur.git/tree/license.txt?h=spooles

Reply:
My guess is that the `Arch` packager made a mistake :-/

Reply:
So how to proceed?

Reply:
So how to proceed?

Reply:
I would go with what is in the official website.

Reply:
Yeah, remove the LGPL references